### PR TITLE
Add missing commands for P128, P135, P146, P148, P152 and P153

### PR DESCRIPTION
**Pull request by tonhuisman**
Add missing commands for:
- P128 - Neopixel (BusFX)
- P135 - SCD4x
- P146 - Cache reader
- P148 - Display POWR3xxD/THR3xxD
- P152 - Dac
- P153 - SHT4x
- Correct some wrongly constructed command sequences for `command,subcmd`

### CODE REVIEW

File: espeasy.js [hunk 1]
```diff
@@ -84,14 +84,26 @@ var commonPlugins = [
   "multirelay", "multirelay,on", "multirelay,off", "multirelay,set", "multirelay,get", "multirelay,loop",
   //P126
   "ShiftOut", "ShiftOut,Set", "ShiftOut,SetNoUpdate", "ShiftOut,Update", "ShiftOut,SetAll", "ShiftOut,SetAllNoUpdate", "ShiftOut,SetAllLow", "ShiftOut,SetAllHigh", "ShiftOut,SetChipCount", "ShiftOut,SetHexBin",
-  //P129
-  "ShiftIn", "ShiftIn,PinEvent", "ShiftIn,ChipEvent", "ShiftIn,SetChipCount", "ShiftIn,SampleFrequency", "ShiftIn,EventPerPin",
   //P127
   "cdmrst",
+  //P128
+  "neopixelfx,", "nfx,off", "nfx,on", "nfx,dim", "nfx,line,", "nfx,hsvline,", "nfx,one,", "nfx,hsvone,", "nfx,all,", "nfx,rgb,", "nfx,fade,", "nfx,hsv,", "nfx,colorfade,", "nfx,rainbow", "nfx,kitt,", "nfx,comet,", "nfx,theatre,", "nfx,scan,", "nfx,dualscan,", "nfx,twinkle,", "nfx,twinklefade,", "nfx,sparkle,", "nfx,wipe,", "nfx,dualwipe", "nfx,fire", "nfx,fireflicker", "nfx,faketv", "nfx,simpleclock", "nfx,stop", "nfx,statusrequest", "nfx,fadetime,", "nfx,fadedelay,", "nfx,speed,", "nfx,count,", "nfx,bgcolor",
```
Comment (chromoxdor) [inline espeasy.js:90]: It is better to also add the primary command. (in this case "nfx") 
With this you can autocomplete the primary command, type a comma and choose the secondary command.
Without this you´ll always autocomplete the whole first entry of primary + secondary command.

File: espeasy.js [hunk 2]
```diff
@@ -84,14 +84,26 @@ var commonPlugins = [
...
+  //P129
+  "ShiftIn", "ShiftIn,PinEvent", "ShiftIn,ChipEvent", "ShiftIn,SetChipCount", "ShiftIn,SampleFrequency", "ShiftIn,EventPerPin",
```
Comment (chromoxdor) [inline espeasy.js:92]: Here it is correct...

File: espeasy.js [hunk 3]
```diff
@@ -84,14 +84,26 @@ var commonPlugins = [
...
+  //P135
+  "scd4x,storesettings", "scd4x,facoryreset", "scd4x,selftest", "scd4x,setfrc,",
```
Comment (chromoxdor) [inline espeasy.js:94]: It´s also missing here and in the rest of the added commands...sorry that´s my fault i never wrote an instruction 🫣

Comment (chromoxdor) [reply espeasy.js:94]: Tomorrow ican correct this here an in the espeasy repo if you want

File: espeasy.js (hunk 1 above)
Comment (tonhuisman) [reply espeasy.js:90]: I'll duplicate the `nfx` range to `neopixelfx` as well, just to be complete. Not sure if it is actually used though...

File: espeasy.js (hunk 3 above)
Comment (tonhuisman) [reply espeasy.js]: I have already made a commit in the ESPEasy repo with the latest changes 😅

**Pull request merged**